src/Parser/Mage/Frost/Modules/Items/Tier20_2set.js

```diff
@@ -5,7 +5,7 @@ import SpellLink from 'common/SpellLink';
 import Combatants from 'Parser/Core/Modules/Combatants';
 import Module from 'Parser/Core/Module';
 import HIT_TYPES from 'Parser/Core/HIT_TYPES';
-import getDamageBonus from '../MageCore/GetDamageBonus';
+import getDamageBonus from 'Parser/Mage/Shared/Modules/GetDamageBonus';
```

Comment:
Both this and Zann'esu Journey could be replaced with relative paths but that doesn't really change anything (and in this case it would make the import path even longer and perhaps less readable) but this should be equal to `../../../Shared/Modules/GetDamageBonus`.

Reply:
Personally, i dont like using relative paths when i have to go backwards more than 1-2 directories, so i usually use absolute paths in that case just because its simpler to see it and know where its pointing as opposed to trying to decipher a relative path that goes back a bunch of directories. But thats just me. But when its in the same directory or one back, then i dont have a problem using relative paths.

Reply:
I agree, just wanted to point out the possibility :-)